### PR TITLE
Configuration of DRclusters to enable fencing

### LINKED ISSUE #7902: On MDR deployments, dr clusters are not configured for fencing
This section needs to be configured: https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/configuring_openshift_data_foundation_disaster_recovery_for_openshift_workloads/metro-dr-solution#configure-drclusters-for-fencing-automation

### PR DESCRIPTION
**Pull request by parikshithb**
fixes #7902 and #7901

### CODE REVIEW

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -2286,6 +2288,9 @@ def deploy_dr_multicluster_orchestrator(self):
             condition="1", column="AVAILABLE", resource_count=1, timeout=600
         )
 
+        # Enable MCO console plugin
+        enable_mco_console_plugin()
```
Comment (prsurve) [inline ocs_ci/deployment/deployment.py:2292]: This should be done only for MDR for RDR we already have console plugin enabled via operator install

File: ocs_ci/deployment/deployment.py [hunk 2]
```diff
@@ -2654,6 +2659,8 @@ def deploy(self):
         self.configure_mirror_peer()
         # Deploy dr policy
         self.deploy_dr_policy()
+        # Configure DRClusters for fencing automation
+        configure_drcluster_for_fencing()
```
Comment (prsurve) [inline ocs_ci/deployment/deployment.py:2663]: This should be done only for MDR

Comment (parikshithb) [reply ocs_ci/deployment/deployment.py:2662]: This is done inside MDR class [2644](https://github.com/red-hat-storage/ocs-ci/pull/7978/files#diff-8571fc52f1040ac2cb01985da6d61cc93a535f812814c200b45d00daf28c2e3fL2639)

File: ocs_ci/deployment/deployment.py (hunk 1 above)
Comment (parikshithb) [reply ocs_ci/deployment/deployment.py]: Moved inside MDR

File: ocs_ci/deployment/deployment.py (hunk 2 above)
Comment (prsurve) [reply ocs_ci/deployment/deployment.py:2662]: Okay

**Comment by parikshithb** [comment]
verified locally here: http://pastebin.test.redhat.com/1104631

**Review by Akarsha-rai — DISMISSED** [review]

**Comment by parikshithb** [comment]
validated again here: http://pastebin.test.redhat.com/1105309

**Review by Akarsha-rai — DISMISSED** [review]

File: ocs_ci/helpers/dr_helpers.py [hunk 3]
```diff
@@ -612,6 +617,39 @@ def get_all_drpolicy():
     return drpolicy_list
 
 
+def get_managed_cluster_node_ips():
+    """
+    Gets node ips of individual managed clusters for enabling fencing on MDR DRCluster configuration
+    Returns:
```
Comment (petr-balogh) [inline ocs_ci/helpers/dr_helpers.py:623]: There should be one empty line before the returns section

**Review by petr-balogh — DISMISSED** [review]

File: ocs_ci/helpers/dr_helpers.py (hunk 3 above)
Comment (parikshithb) [reply ocs_ci/helpers/dr_helpers.py:624]: ack, made the change

**Review by Akarsha-rai — APPROVED** [review]

**Pull request merged**